sqlmesh/migrations/v0026_fix_view_kind_data_hash.py

```diff
@@ -0,0 +1,5 @@
+"""Reflect the `materialized` flag of the VIEW model kind in the data hash."""
+
+
+def migrate(state_sync):  # type: ignore
+    pass
```

Comment:
Just FMI: what does an empty migration achieve?

Reply:
It triggers the fingerprint recomputation. I'm going to drop it because I'm going to have a real migration script in a different PR.

Reply:
Gotcha, thanks for explaining